library/general/src/lib/installation/proposal_client.rb

```diff
@@ -132,6 +137,11 @@ def run
     #     The caller will format each item as a HTML list item (`<li>`). The
     #     proposal can contain hyperlinks with IDs listed in the list `links`.
     #
+    #   * **`"label_proposal"`** [Array<String>, nil]
```

Comment:
OK, I see your point. The result already has 10 keys (not fully clear which are optional). But IMHO reusing `raw_proposal` would complicate both the implementation and the documentation:

> `raw_proposal`: Human readable proposal, not formatted yet. For `cramped_label_mode` they will be put in Labels. Otherwise the caller will format each item as a HTML list item (`<li>`). The proposal can contain hyperlinks with IDs listed in the list links.